cpp/missing_metadata_db_tool.cc

```diff
@@ -1,4 +1,4 @@
-/** \brief Utility for finding oerforming certain edits on the metadata_presence_tracer MySQL table in ub_tools.
+/** \brief Utility for finding performing certain edits on the metadata_presence_tracer MySQL table in ub_tools.
```

Comment:
as discussed, please remove this tool entirely & restore validate_harvested_records.cc

Reply:
Done